addons/new-osio-addon/src/test/java/io/fabric8/launcher/osio/steps/OpenShiftStepsTest.java

```diff
@@ -82,10 +76,17 @@ public void shouldCreateBuildConfig() throws IOException, URISyntaxException {
                 .build();
         TokenIdentity identity = IdentityFactory.createFromToken("123");
         List<ImmutableNamespace> elements = Collections.singletonList(namespace);
-        openshiftClient.tenant = ImmutableTenant.builder().identity(identity)
+        Tenant tenant = ImmutableTenant.builder().identity(identity)
                 .username("edewit").email("me@nerdin.ch").namespaces(elements).build();
 
-        steps.tenant = openshiftClient.tenant;
+        String openShiftApiURL = EnvironmentVariables.getOpenShiftApiURL();
+        Config config = new ConfigBuilder().withMasterUrl(openShiftApiURL).withOauthToken("123")
+                .withTrustCerts(true).build();
+
+        OpenShiftService openShiftService = new OpenShiftService(new DefaultKubernetesClient(config), tenant);
+        steps.openShiftService = openShiftService;
```

Comment:
This looks like a code smell. Is this because Arquillian is too slow to run this test?